account-kit/smart-contracts/src/light-account/utils.ts

```diff
@@ -326,36 +324,3 @@ export async function getLightAccountVersionDef<
 
   return AccountVersionRegistry[accountType][version];
 }
-
```

Comment:
Hmm this seems like it is out of scope of this PR?

Reply:
In the future would recommend we break this stuff out. It makes the revert-ability, plus muddies up the history.

Reply:
oh yea that's my bad, think I deleted it and just forgot to break it out 🤦